src/checkout-buttons/checkout-button-options.ts

```diff
@@ -21,7 +21,7 @@ export interface CheckoutButtonInitializeOptions extends CheckoutButtonOptions {
      * The options that are required to facilitate AmazonMaxo. They can be
      * omitted unless you need to support AmazonMaxo.
      */
-    amazonmaxo?: AmazonMaxoButtonInitializeOptions;
+    amazonpay?: AmazonMaxoButtonInitializeOptions;
```

Comment:
The name of this object `AmazonMaxoButtonInitializeOptions`, together with the comment above will get exposed in the sdk documentation. It is probably a good idea to change these.

Reply:
Oops, yeah missed this bit. Maybe search and replace `Maxo` with `Pay` in the entire codebase to be safe.

Reply:
We have many class names and file names with AmazonMaxo because `AmazonPay` was used for amazon v1! if I change everything to `amazonpay` it will be even more confusing because v1 is also called `amazonPay` for v1 except the methodId which is just  `amazon`. The other option is rename all to amazonPayv2 but the method id must be `amazonpay` @icatalina @capsula4

Reply:
Discussed offline. We are renaming `AmazonMaxoButtonInitializeOptions` to `AmazonPayButtonInitializeOptions`. A follow up PR will rename all the internal objects from `AmazonMaxo...` to `AmazonPayV2...`.